lunes_cms/cms/forms.py

```diff
@@ -26,6 +26,34 @@ def label_from_instance(self, obj):
             return obj.title
 
 
+class DocumentForm(forms.ModelForm):
+    """
+    This is the form for the Document model
+    """
+
+    class Meta:
+        """
+        Defining the meta description for the `DocumentForm`
+        """
+
+        model = Document
+        fields = ["word_type", "plural", "plural_article"]
+
+        def clean(self):
+            word_type = self.clean_fields.get("word_type")
+
+            if word_type == "Substantiv":
+                self.fields_required(["plural"])
+            else:
+                self.clean_fields["plural"] = ""
```

Comment:
```suggestion
    def clean(self):
        word_type = self.cleaned_data.get("word_type")

        if word_type == "Nomen":
            self.fields_required(["plural", "plural_article"])
        else:
            self.cleaned_data["plural"] = ""
            self.cleaned_data["plural_article"] = None
```
1. The indentation is incorrect, this method belongs into the `DocumentForm` class and not the inline `Meta` class.
2. The database-value of Substantives are "Nomen": https://github.com/digitalfabrik/lunes-cms/blob/03d6236a57a462aa30b4f445776ff8872a21c451/lunes_cms/cms/models/static.py#L24
3. `clean_fields` does not exist, the attribute is called `cleaned_data`.
4. I think we should enforce the same rules on `plural_article` as well.

Reply:
I see, thank you for the detailed explaination!